openupgrade_scripts/scripts/website_slides/15.0.2.4/post-migration.py

```diff
@@ -0,0 +1,7 @@
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    # Load noupdate changes
+    openupgrade.load_data(env.cr, "website_slides", "15.0.1.0/noupdate_changes.xml")
```

Comment:
```suggestion
    openupgrade.load_data(env.cr, "website_slides", "15.0.2.4/noupdate_changes.xml")
```

@sang250399 Phải thế này chứ nhỉ. Có test thử chưa đó em.

Reply:
@tranngocson1996 dạ vâng để e sửa lại ạ.